Terran-One/cw-simulate-ui
# Removed Execution History and moved everything to trace.

## Issue link

[WL-XXX](https://trello.com/c/IjDe0cRG/15-remove-execution-history-and-use-trace-instead)

## Description
State Stepper was earlier using Execution History, now everything is moved to trace and we don't store any state in UI anymore.

## Test steps

1. Go to simulation page.
2. Perform some executions.
3. Make sure stepper is working correctly.
